### PR TITLE
Add support for BeagleBone Green Gateway

### PR DESCRIPTION
**Pull request by jolesieur**
Added eeprom ID for the new BBG Gateway from Seeed Studio.

### CODE REVIEW

File: adafruit_platformdetect/constants/boards.py [hunk 1]
```diff
@@ -250,6 +252,7 @@
     BEAGLEBONE_POCKETBEAGLE: (("A2", "A335PBGL00A2"),),
     BEAGLEBONE_GREEN: (("1A", "A335BNLT...."), ("UNKNOWN", "A335BNLTBBG1")),
     BEAGLEBONE_GREEN_WIRELESS: (("W1A", "A335BNLTGW1A"),),
+    BEAGLEBONE_GREEN_GATEWAY: (("A335BNLTGG1A"),),
```
Comment (ladyada) [inline adafruit_platformdetect/constants/boards.py:255]: this should be a tuple like the ones above?

**Comment by ladyada** [comment]
ok thanks! please run `black` on the files to format it

File: adafruit_platformdetect/constants/boards.py (hunk 1 above)
Comment (jolesieur) [reply adafruit_platformdetect/constants/boards.py]: Yeah, My bad. I've made the correction to BEAGLEBONE_GREEN_GATEWAY: (("G1A", "A335BNLTGG1A"),),

**Review by jolesieur — COMMENTED** [review]
Unable to complete code verification.

**Comment by makermelissa** [comment]
It appears you completely deleted board.py in this PR, so it's failing.

**Pull request closed without merging**